src/services/utils.js

```diff
@@ -0,0 +1,4 @@
+const generateUniqeUsername = (email) =>
```

Comment:
lets always include curly brackets to function even if its 1 line

Reply:
Okay